Sources/SwiftInspectorVisitors/TypeConformanceVisitor.swift

```diff
@@ -0,0 +1,41 @@
+// Created by Dan Federman on 1/27/21.
+//
+// Copyright © 2021 Dan Federman
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Foundation
+import SwiftSyntax
+
+public final class TypeConformanceVisitor: SyntaxVisitor {
+
+  public private(set) var conformsToTypes = [String]()
+
+  public override func visit(_ node: InheritedTypeSyntax) -> SyntaxVisitorContinueKind {
+    conformsToTypes.append(node.typeName.description.trimmingCharacters(in: .whitespacesAndNewlines))
+    return .visitChildren
+  }
+
+  public override func visit(_ node: MemberDeclBlockSyntax) -> SyntaxVisitorContinueKind {
+    // We don't care about the internals of this type.
```

Comment:
Is that true for this use case? Shouldn't we care here?

Reply:
What clause would this visitor care about within this code block?

As an example, say we're using this type inside of a `ClassDeclSyntax` block to find the information about a class definition. We don't care about what's in the class itself, we just care about the declaration. If there's another class defined within this class, it's up to the parent visitor to continue walking the tree and then invoke this visitor again if we run into another class.

Reply:
Gotcha! That makes total sense! Thanks!